targets/XPlatCppSdk/make.js

```diff
@@ -287,9 +289,11 @@ function getRequestActions(tabbing, apiCall, isInstanceApi) {
             + tabbing + "    else if (" + authContext + "clientSessionTicket.length() > 0) {\n"
             + tabbing + "        authKey = \"X-Authorization\"; authValue = " + authContext + "clientSessionTicket;\n"
             + tabbing + "    }\n"
+            + tabbing + "    #if defined(ENABLE_PLAYFABSERVER_API) || defined(ENABLE_PLAYFABADMIN_API)\n"
```

Comment:
>     [](start = 25, length = 4)

Please remove these spaces, we usually put preprocessor directives at the very beginning of a line